### PR TITLE
Payee 293 cy pension details

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/IncomeSourceSummaryView.scala.html [hunk 1]
```diff
@@ -156,69 +172,89 @@ <h2 class="flush--top">@messages("tai.taxCode")</h2>
...
-                    <span aria-hidden="true">@pounds(benefit.amount)</span>
-                    <span class="visuallyhidden">@Html(messages("tai.income.details.benefit.amount.announce", withPoundPrefixAndSign(MoneyPounds(benefit.amount, 0))))</span>
-                </div>
-                <div class="tai-description-details--retain-column" id="companyBenefitChangeLinkDescription@idx">
+    <div class="govuk-!-padding-bottom-7"></div>
+    <div class="govuk-grid-row">
+        <div id="companyBenefitsSection" class="govuk-!-padding-left-3">
+            <h2 id="companyBenefitsHeading" class="govuk-heading-m ">@Messages("tai.income.details.companyuBenefitsHeading", model.empOrPensionName)</h2>
+            @if(model.benefits.size > 0) {
+                <table id="companyBenefitList" class="govuk-table responsive-table-fix">
```
Comment (vivekhmrc) [inline app/views/IncomeSourceSummaryView.scala.html:194]: This responsive-table-fix seems to have done the trick 👍

File: app/views/IncomeSourceSummaryView.scala.html [hunk 2]
```diff
@@ -60,24 +60,38 @@
...
-@mainTemplate(
+@main(
     title = title,
+    pageTitle = Some(Messages("tai.service.navTitle")),
     authedUser = Some(request.taiUser),
-    pageHeader = Some(header),
-    articleClasses = None
+    pagePrintable = true,
+    backLinkContent = Messages("return.to.your.income.tax.summary"),
+    backLinkUrl = Some(routes.TaxAccountSummaryController.onPageLoad.url),
```
Comment (vivekhmrc) [inline app/views/IncomeSourceSummaryView.scala.html:81]: This looks good for putting the URL of the back link 👍

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

**Review by Jamie22Butler — APPROVED** [review]
LGTM 👍

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

**Pull request merged**